todo_app/data/session_items.py

```diff
@@ -1,20 +1,35 @@
-from flask import session
+from flask import Flask
+from requests import get, post, put
+from todo_app.flask_config import Config
+from todo_app.data.ToDoItem import ToDoItem 
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
+app = Flask(__name__)
+app.config.from_object(Config)
 
+# Constants
+_BASE_PARAMS = {'key': app.config['TRELLO_KEY'], 'token':app.config['TRELLO_TOKEN']}
+_BOARD_BASE_URL = 'https://api.trello.com/1/boards/GjbQrcgh/cards'
+_CARDS_BASE_URL = 'https://api.trello.com/1/cards'
 
-def get_items():
+
+# Creating this local storage as session data must be serialisable which messes up with class methods
+class LocalItemList:
```

Comment:
Rather than pre-emptively trying to cache this list, I'd be inclined to leave it as always freshly determined for now; if we find this is an issue then we can add caching at a later date

Reply:
Removed cached version in commit 4f9b126